system-tests/lib/cre/environment/environment.go

```diff
@@ -179,10 +177,27 @@ func SetupTestEnvironment(
 	if topoErr != nil {
 		return nil, pkgerrors.Wrap(topoErr, "failed to build topology")
 	}
-
 	fmt.Print(libformat.PurpleText("%s", input.StageGen.WrapAndNext("DONs configuration prepared in %.2f seconds", input.StageGen.Elapsed().Seconds())))
-	fmt.Print(libformat.PurpleText("%s", input.StageGen.Wrap("Starting Job Distributor and DONs and linking them to JD")))
 
+	fmt.Print(libformat.PurpleText("%s", input.StageGen.Wrap("Applying Features before DON startup")))
+	_, preErr := operations.ExecuteOperation(deployKeystoneContractsOutput.Env.OperationsBundle, PreDONStartupOp, PreDONStartupOpDeps{
```

Comment:
@Tofel IMO it's overkill to create an Op here; we are never going to use these ops in CLD proper, so let's not tie ourselves to this API